tests/unit/components/taskItem.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import TaskItem, { ITodoItemProps } from '@app/components/tasks/TaskItem';
+import { getProps } from './propsTaskItem';
+
+let props: ITodoItemProps;
+
+beforeEach(() => {
+  props = getProps();
```

Comment:
Usually we have `renderComponet` function and don't need `getProps`. For example 
```javascript
import TaskItem from '@app/components/tasks/TaskItem';

const mockTodo: Todo = {
  id: '2',
  name: 'Make task',
  isDone: false,
  isEdit: false,
  date: formatDate(new Date()),
};

const renderComponent = () => render (
  <TaskItem
      key={mockTodo.id}
      todo={mockTodo}
      changeEditedTask={jest.fn()}
      toggleEditTask={jest.fn()}
      toggleCompleteTask={jest.fn()}
      removeTask={jest.fn()}
   />
}
```

and in test 

```javascript
const { getByTestId, getByText } = renderComponent();
```